pkg/inspect/inspect.go

```diff
@@ -0,0 +1,77 @@
+package inspect
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/square/p2/pkg/health"
+	"github.com/square/p2/pkg/kp"
+)
+
+const (
+	INTENT_SOURCE = iota
+	REALITY_SOURCE
+)
+
+type NodePodStatus struct {
+	NodeName           string             `json:"node,omitempty"`
+	PodId              string             `json:"pod,omitempty"`
+	IntentManifestSHA  string             `json:"intent_manifest_sha"`
+	RealityManifestSHA string             `json:"reality_manifest_sha"`
+	IntentLocations    []string           `json:"intent_locations"`
+	RealityLocations   []string           `json:"reality_locations"`
+	Health             health.HealthState `json:"health,omitempty"`
+}
+
+type NodeHealth struct {
```

Comment:
Sorry, I should have caught this earlier.  It looks like this structure is never used. Maybe now, during a refactor, is a good time to get rid of it. Can you amend the commit?